cross/c_icap/Makefile

```diff
@@ -1,8 +1,8 @@
 PKG_NAME = c_icap
-PKG_VERS = 0.2.2
+PKG_VERS = 0.3.5
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://downloads.sourceforge.net/project/c-icap/c-icap/0.2.x
+PKG_DIST_SITE = http://downloads.sourceforge.net/project/c-icap/c-icap/0.3.x
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
```

Comment:
Compiling fails for a number of toolchains, and consequently, compiling the package fails:
```
88f5281-3.1
x86/64 4.3 (bromolow, cedarview, x86)
powerpc-4.0
ppc824x-4.2
ppc853x-4.3
ppc854x-4.0
```
If there's a quickfix for the failures, please do so, otherwise add constraints by setting `REQUIRED_DSM` and `UNSUPPORTED_ARCHS` in the Makefile.